source/SquadLeader/.idea/encodings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I did research this and put everything in [.gitignore](https://github.com/Esri/squad-leader-android/blob/master/source/SquadLeader/.gitignore) that is recommended to be there. The .idea folder helps make it an Android Studio project. Android Studio is based on IntelliJ IDEA, hence the .idea name.